examples/tensorflow/train_predict.py

```diff
@@ -2,53 +2,69 @@
 from __future__ import division
 from __future__ import print_function
 
+import argparse
 import mlflow
 from mlflow import tensorflow, tracking, pyfunc
 import numpy as np
 import pandas as pd
 import shutil
 import tempfile
 import tensorflow as tf
+import os.path
 
+# Command-line arguments
+parser = argparse.ArgumentParser(description='Tensorflow Example')
+# parser.add_argument('--hidden_units', nargs='*', default='50 20'.split( ), metavar='L',
+#                    help='hidden units (default: [50, 20])')  #  https://docs.python.org/3/library/argparse.html#nargs
+parser.add_argument('--hidden_units', type=str, default='50 20', metavar='L',
+                    help='hidden units (default: "50 20")')
```

Comment:
Nit: Instead of copying the default "50 20" value, use `%(default)s`. Let's also make the help text a little more descriptive. The changed line will look like

```
help='The number of hidden units in each of the two hidden unit layers (default: %(default)s)
```

For reference about this behavior, see https://stackoverflow.com/a/18507871